bundle.yaml

```diff
@@ -10190,6 +10190,118 @@ spec:
                   pageTitle:
                     description: The prometheus web page title
                     type: string
+                  tlsConfig:
```

Comment:
Could you move generated content into separate PR? It will help with the review.

Reply:
Should I add the generated assets as a new commit or a new PR? Wouldn't a new PR break atomic changes which are handy in case of rollbacks?

Reply:
As a separate commit in the same PR.

Reply:
Roger. Commits should be now split into written and auto-generated code.